plaso/lib/errors.py

```diff
@@ -27,6 +27,8 @@ class HeapFull(Error):
 class InvalidEvent(Error):
   """Error indicating an event is malformed."""
 
+class InvalidFilter(Error):
```

Comment:
Should we specify the type of filter here? Seeing plaso has many different types of filters.

Reply:
I don't think so - we use pretty exceptions generic elsewhere in the codebase. We can change this to a more specific name if we need to disambiguate between different filters later.